src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const cssObject = {};
+
+  const noSpacesArray = sourceString.split(';');
+  const arrayOfArrays = noSpacesArray
+    .map((element) => element.split(':'))
+    .filter((element) => element.length === 2)
+    .map((element) => [element[0].trim(), element[1].trim()]);
```

Comment:
To improve code readability, ensure that all chained methods are indented at the same level.